shapely/tests/test_io.py

```diff
@@ -287,7 +293,7 @@ def test_to_wkt_exceptions():
         shapely.to_wkt(1)
 
     with pytest.raises(shapely.GEOSException):
-        shapely.to_wkt(point, output_dimension=4)
+        shapely.to_wkt(point, output_dimension=5)
```

Comment:
GEOS now allows to specify an output dimension of 4, for the case your geometry is value 4. We don't yet have ways to create such geometries on our side, but so changing the value here to trigger the same error as before on GEOS side.